### ISSUE TITLE
Are Kickflip services reliable for long term?

### ISSUE THREAD

**Issue by omarojo** [Kickflip/kickflip-docs#6]
I see Kickflip is a great online service, I tried the demo app and works great. But I dont see any contact info with the developers or customer service. Feels like a startup in very early stage, so Im not very confident to port the huge amount of users I have under your services. 

It would be great if you give an update on the status of Kickflip as a service or company so that it doesnt end up like http://blog.cine.io/ which was a super cool and robust service that is now dead.

**Comment by Miserlou** [comment]
We're in it it to win it and intend to keep Kickflip online for the foreseeable future.

Full disclosure though, we are a very small team and a bootstrapped company. However, we are profitable enough from sales and consulting that we are able to keep the service comfortably online. Our biggest problem is being responsive to support requests, but that's the thing that we'll be addressing next.

Thanks for your interest in Kickflip!

**Comment by Miserlou** [comment]
Actually, if you want to talk more about the specifics of your situation, we should probably schedule a call to go over it. We have a hosted plan that may suit your needs best, or we can do custom integration.

Send me a message at rich@gun.io?

**Comment by omarojo** [comment]
Thanks for reply, I will send you an email to talk more about it.

**Comment by omarojo** [comment]
Hi Rich I sent you an email to rich@gun.io, did you receive it ?

**Comment by alaxicsmith** [comment]
@omarojo did this end up panning out for you?

**Comment by omarojo** [comment]
Hi Alex,

I sent several emails to your support team. But never received a reply. We are heavily considering using Kickflip but there are concerns regarding your SDK and specially your pricing tiers.

**Comment by alaxicsmith** [comment]
@omarojo I don't work for Kickflip. I'm considering using them for my own needs, but I'm unsure of how stable of a platform this is.

**Comment by omarojo** [comment]
Well as you can see, its been almost a month without a reply from them on any channel.
I might still use their system for prototyping, because it offers great solution. But its hard to know how reliable they are.

-- 
Ing. Omar Juárez

On October 6, 2015 at 1:51:01 PM, Alaxic Smith (notifications@github.com) wrote:

@omarojo I don't work for Kickflip. I'm considering using them for my own needs, but I'm unsure of how stable of a platform this is.

**Comment by guyverrtsb** [comment]
@omarojo...have you had any luck with the responses?  I am also wanting to use KickFlip unique solution to HLS within our service layers but am not able to get in touch with anyone.  we have also asked for a consulting help to deliver exactly what we need that we would pay for with no luck.  

I have been cheering for this team and really hope they are doing well.  

@KickFlip...if you could respond we would greatly appreciate doing work with you to help keep these moving forward delivering a strong relationship.

**Comment by omarojo** [comment]
@guyverrtsb Nope, I gave up on Kickflip. I decided to get in contact with http://www.panda-os.com they have a Video Management system, and mobile capabilities similar to Kickflip, but you need to contact them to have access to their solutions. Also there is Red5Pro and the mobile SDK.
